Update zha.markdown with ZHA remove service example (#17457)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

### Service `zha.remove`

> This "Zigbee Home Automation: remove" service removes an existing device from the network. IEEE address of the device can be found on the device card of Zigbee devices, (example of IEEE address data parameter format is `00:0d::6f:00:05:7d:2d:34`).

Reviewer: ```suggestion
This service removes an existing device from the network. You can find the IEEE address of the device on the device card of Zigbee devices. An example of an IEEE address data parameter format is `00:0d::6f:00:05:7d:2d:34`.
```

|  Data | Optional | Description |
| ---- | ---- | ----------- |

> | `ieee_address` | no | IEEE address of the device to remove

Reviewer 2: Leave this one as is. Services.yaml needs to be updated. `ieee_address` was deprecated toward `ieee`

Author: OK
